package.json

```diff
@@ -34,7 +34,7 @@
     "prebuild": "rimraf dist tmp",
     "prebuild:watch": "rimraf dist tmp",
     "preversion": "npm run ci",
-    "test": "ng test --code-coverage --no-watch --no-progress --browsers=ChromeHeadlessCI --source-map=true",
+    "test": "ng test --code-coverage --no-watch --no-progress --browsers=ChromeHeadlessCI",
```

Comment:
@johnbenjamin-hmcts - the `--source-map=true` bit has been removed...?

Reply:
Hi @Daniel-Lam , yes had to remove due to the following error:
PS C:\HMCTS\ccd-case-ui-toolkit> yarn test
yarn run v1.22.11
$ ng test --code-coverage --no-watch --no-progress --browsers=ChromeHeadlessCI --source-map=true
Schema validation failed with the following errors:
Data path ".sourceMap" should be object.
Data path ".sourceMap" should be boolean.
Data path ".sourceMap" should match exactly one schema in oneOf.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

The tests are running fine without this param, and MC doesn't have this defined as well. So I had removed it.

Reply:
@johnbenjamin-hmcts - as per PR #967, that's fine.